.travis.yml

```diff
@@ -30,16 +30,12 @@ matrix:
     - python: 2.7
     - python: 3.4
     - python: 3.5
-  allow_failures:
-    - python: 2.7
 before_install:
-  - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
   - pip install -U pip
   - pip install wheel
 install:
   - travis_wait pip wheel -r ci/requirements.txt
-  - travis_wait pip install -r ci/requirements.txt
+  - travis_wait pip install --upgrade --force-reinstall -r ci/requirements.txt
```

Comment:
It still uses the wheels, just forces installation even if the same version is already installed(by the system package manager in this case).